packages/@guardian/source-foundations/src/colour/palette.stories.mdx

```diff
@@ -2,11 +2,11 @@ import { Meta } from '@storybook/addon-docs';
 import { palette } from './palette';
 
 <Meta
-	title="Packages/source-foundations/palette"
-	parameters={{ previewTabs: { canvas: { hidden: true } } }}
+	title="Packages/source-foundations"
+	parameters={{ previewTabs: { canvas: { hidden: true } }}}
 />
 
-# `palette`
+# Palette
```

Comment:
since `palette` is a named export (not just a concept), we wanted to convey that by styling it as code. did it break somtehing?

Reply:
I agree, and it should ideally be reflected as such the menu/heading as lowercase.